authority-portal-frontend/src/styles.css

```diff
@@ -279,3 +279,10 @@ select:-webkit-autofill:disabled {
 .modified-tooltip > .mdc-tooltip__surface {
   max-width: 400px !important;
 }
+
+.roles-tooltip > .mdc-tooltip__surface {
+  max-width: 400px !important;
+  max-height: unset !important;
+  padding: 1rem !important;
+  white-space: pre-wrap !important;
+}
```

Comment:
why does the screenshot not show the tooltip at the application role select.

if we have these tooltips, we should have them everywhere

Reply:
done